tests/phpunit/class-wc-stripe-api-sdk-test.php

```
@@ -0,0 +1,506 @@
+			->method( 'retrieve' )
+			->with( $source_id )
+			->willReturn( $response );
+
+		$this->inject_sdk( $this->make_sdk_with_service( 'sources', $mock_sources ) );
+
+		$result = WC_Stripe_API::get_payment_method( $source_id );
+
+		$this->assertInstanceOf( \Stripe\StripeObject::class, $result );
+		$this->assertEquals( $source_id, $result->id );
+		$this->assertEquals( 'source', $result->object );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_payment_method_retrieves_source is not named in camelCase. (undefined)

(CamelCaseMethodName)